app/src/main/java/com/android/unio/model/association/Association.kt

```diff
@@ -104,9 +109,10 @@ class Role(val displayName: String, val permissions: Permissions, override val u
                 .addPermission(PermissionType.EDIT_MEMBERS)
                 .addPermission(PermissionType.VIEW_EVENTS)
                 .build(),
+            0xFF0000FF,
             "Committee")
-    val MEMBER = Role("Member", Permissions.NONE, "Member")
-    val GUEST = Role("Guest", Permissions.NONE, "Guest")
+    val MEMBER = Role("Member", Permissions.NONE, badgeColorBlue, "Member")
+    val GUEST = Role("Guest", Permissions.NONE, badgeColorBlue, "Guest")
```

Comment:
Why did you assign the same color to all of the predefined roles ? I think it would be more useful to assign a different color to each of them.

Reply:
And it would improve my line coverage ; ) Sure